src/httpServer/handler.ts

```diff
@@ -23,6 +23,8 @@ type HandlerWfFuncs<T> = {
 export interface HandlerContext extends DBOSContext {
   readonly koaContext: Koa.Context;
   invoke<T extends object>(targetClass: T, workflowUUID?: string): InvokeFuncs<T>;
+  startWorkflow<T extends object>(targetClass: T, workflowUUID?: string): InvokeFuncs<T>;
+  runWorkflow<T extends object>(targetClass: T, workflowUUID?: string): InvokeFuncs<T>;
```

Comment:
I think we should use more explicit names to indicate async vs sync invocations.
For example, `execWorkflowAsync` and `execWorkflowSync`

Reply:
Decided against these as the `async` keyword is meaningful in JS/TS.